src/elastio-snap.c

```diff
@@ -1244,6 +1244,20 @@ static inline void tracer_set_cow_fail_state(struct snap_device *dev, int error)
 	smp_mb();
 }
 
+static inline int tracer_fail_state(struct snap_device *dev){
```

Comment:
@kgermanov unfortunately there is an issue with CI checks on a PR made from a fork. It hasn't been resolved yet. But I did a manual build from your branch. And there is a compilation warning that can be easily resolved:
```
/home/elastio/aaa/elastio-snap/src/elastio-snap.c: In function ‘tracer_elastio_snap_info’:
/home/elastio/aaa/elastio-snap/src/elastio-snap.c:4674:34: warning: passing argument 1 of ‘tracer_fail_state’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
 4674 |  info->error = tracer_fail_state(dev);
      |                                  ^~~
/home/elastio/aaa/elastio-snap/src/elastio-snap.c:1247:57: note: expected ‘struct snap_device *’ but argument is of type ‘const struct snap_device *’
 1247 | static inline int tracer_fail_state(struct snap_device *dev){

```

Cold you please do fix this warning? And I'll mere the PR after that.
```suggestion
static inline int tracer_fail_state(const struct snap_device *dev){
```

Reply:
Yes, of course.